test/integration/alerting/install.yaml

```diff
@@ -0,0 +1,50 @@
+application:
+  validators:
+    - name: cosign
+      type: cosign
+      trustRoots:
+        - name: default
+          key: |
+            -----BEGIN PUBLIC KEY-----
+            MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvtc/qpHtx7iUUj+rRHR99a8mnGni
+            qiGkmUb9YpWWTS4YwlvwdmMDiGzcsHiDOYz6f88u2hCRF5GUCvyiZAKrsA==
+            -----END PUBLIC KEY-----
+
+  policy:
+    - pattern: "securesystemsengineering/testimage:co-*"
+      validator: cosign
+      with:
+        verifyInTransparencyLog: false
+
+alerting:
+  clusterIdentifier: "minikube"
```

Comment:
why this?

Reply:
copied from previous integration test 🤷 changed to `"integration-test-cluster"`